mne/gui/_coreg_gui.py

```diff
@@ -159,6 +180,38 @@ class CoregModel(HasPrivateTraits):
     points_eval_str = Property(depends_on='point_distance')
 
     @cached_property
+    def _get_points(self):
+        with warnings.catch_warnings(record=True):  # comp to None in Traits
+            if self.points_filter is None:
+                self.points_filter = \
+                    np.ones(len(self.hsp.inst_points), np.bool8)
```

Comment:
I thought of that too (`np.logical_and`), but consider this:
* load a hsp source, EEG enabled by default
* disable EEG
* try to Omit by distance

Now when `omit_hsp_points` gets `self.transformed_hsp_points`, `_get_points` returns _fewer points_ (because the EEG points are missing) than are defined in the `points_filter`. Then this line fails due to the mismatch in tested points (`new_sub_filter`) vs. those defined to be True in the `old_filter`:

```
new_filter[old_filter] = new_sub_filter
```

Making `omit_hsp_points` aware of the EEG selector state seems clumsy and prone to future bugs. If I do modify the filter in `_get_points`, though, I end up not being able to turn the EEG back on after they've been left out once (the logical and will forever be False).